content/en/logs/guide/log-collection-troubleshooting-guide.md

```diff
@@ -253,7 +253,15 @@ See the [Lambda Log Collection Troubleshooting Guide][10]
 
 ## Unexpectedly dropping logs
 
-Check if logs appear in the [Datadog Live Tail][11]. If they appear in the Live Tail, check the Indexes configuration page for any [exclusion filters][12] that could match your logs.
+Check if logs appear in the [Datadog Live Tail][11].
+
+If they appear in the Live Tail, check the Indexes configuration page for any [exclusion filters][12] that could match your logs.
+If they do not appear in the Live Tail, they might get dropped as their timestamp it outside the 18 hours ingestion window. You can check which `service` and `source` are impacted with the `datadog.estimated_usage.logs.drop_count` metric.
```

Comment:
```suggestion
If they do not appear in the Live Tail, they might get dropped as their timestamp is outside the 18 hours ingestion window. You can check which `service` and `source` are impacted with the `datadog.estimated_usage.logs.drop_count` metric.
```

Reply:
Also looking at the doc we are saying 

> Log events can be submitted with a [timestamp](https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=date#date-attribute) that is up to 18h in the past.

But we don't talk about the future though so not sure if we want to be explicit on the fact that it's 18h in the past but less for the future

Reply:
Discussed on Slack, and we agreed not to mention the threshold for events in the future. (12 hours for reference)